packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -1132,6 +1137,91 @@ function commitLayoutEffectOnFiber(
   }
 }
 
+function abortParentMarkerTransitions(
+  deletedFiber: Fiber,
+  nearestMountedAncestor: Fiber,
+  abort: TransitionAbort,
+) {
+  let fiber = deletedFiber;
+  let isInDeletedTree = true;
+  while (fiber !== null) {
+    const instance = deletedFiber.stateNode;
```

Comment:
This can be hoisted outside of the loop. Also make the name more specific since there's a marker instance variable below, too.

Reply:
The type should also be refined. What kind of instance is it?